README.md

```diff
@@ -6,7 +6,16 @@ This action builds a [ROS, or ROS 2](https://index.ros.org/doc/ros/) workspace f
 
 ## Requirements
 
-This action requires ROS development tools (`colcon`, `rosdep`, `vcs`) to be installed on the CI worker instance.
+This action requires the following ROS development tools to be installed (and initialized if applicable) on the CI worker instance:
+
+```
+curl
+colcon-common-extensions
+colcon-lcov-result  # Optional
```

Comment:
Not optional.

Reply:
I put "Optional" because this action explicitly ignores the return code of `colcon lcov-result` invocation and doesn't warn the user when it's not installed or when such invocations fail. Example [here](https://github.com/mikaelarguedas/sros2/runs/541492974?check_suite_focus=true)

Which in itself is great for me as I'm using this on action on python packages (so lcov irrelevent), and that `colcon-lcov-result` is not available as a deb so having to install it would be inconvenient

Reply:
I see, got it. It's nice it's working now, but it's an oversight on our side, more than a featue. Adding this to the doc is making it a commitment on our end. We may make this optional, but it needs some consideration outside of the scope of this PR IMHO.